docs/_source/_common/tabs/unfication_strategies.md

```diff
@@ -0,0 +1,55 @@
+Use a `Feedbackdataset` directly:
+
+```python
+from argilla import FeedbackDataset
+
+dataset = Feedbackdataset(...)
```

Comment:
If initialized like this we won't have any response, so feel free to re-direct users either to an existing Argilla dataset in HugginFace via `from_huggingface` or to a placeholder with either `from_huggingface` and/or `from_argilla` instead

Reply:
We discussed this today, will see what I can do.